modules/values.tmpl.yaml

```
@@ -503,6 +503,7 @@ traefik:
         regex: apps.${base_domain}
         replacement: apps.${cluster_name}.${base_domain}
   dashboard:
+    enabled: true
```

I prefer false by default

```suggestion
    enabled: false
```

---

I made that to avoid breaking that for people who use it 
But indeed I would prefer a false too

---

No problem, i will do the change for my platforms.
So false by default

---

Change done :smile: